src/dex-ui-components/base/Select/TokenSelector.tsx

```diff
@@ -17,7 +18,7 @@ const TokenSelector = (props: TokenSelectorProps) => {
   return cloneElement(
     <Select
       variant="tokenSymbolSelector"
-      value={value}
+      value={isEmpty(value) ? undefined : value}
```

Comment:
Fixes a bug where the "Token to Select" placeholder text in the token selector disappears when you click the switch tokens button.